server.rb

```diff
@@ -21,6 +21,13 @@ def current_user
     end
   end
 
+  def require_user
+  	unless session[:logged_in_user_id]
+      session[:flash_message] = "You need to log in to see this page"
+      redirect to ("/sign_in")
+    end
```

Comment:
Are you saying it should read: unless current_user
                                               set_message = "You..."?